railties/test/generators/app_generator_test.rb

```diff
@@ -863,15 +827,13 @@ def test_skip_javascript_option_with_skip_js_argument
 
   def test_hotwire
     run_generator_and_bundler [destination_root]
+
+    assert_includes @rails_commands, "turbo:install stimulus:install", "`turbo:install stimulus:install` expected to be called, but wasn't."
```

Comment:
I think I am :+1: on this change because it should be the gem's responsibility to test its installer with various version of Rails, including edge Rails.  But are we sure there is sufficient test coverage in the `turbo-rails` and `stimulus-rails` gems?  Or are they currently relying on the Rails test suite to act as a canary?

Also, if we're not actually running the installers, then running Bundler is no longer necessary, and we should change `run_generator_and_bundler` to `run_generator` for performance.

Reply:
> I think I am 👍 on this change because it should be the gem's responsibility to test its installer with various version of Rails, including edge Rails. But are we sure there is sufficient test coverage in the `turbo-rails` and `stimulus-rails` gems? Or are they currently relying on the Rails test suite to act as a canary?

I just took a brief look through `turbo-rails` and `stimulus-rails` repos and saw no sort of test, I did find hotwired/turbo-rails#494 and hotwired/stimulus-rails#125 which added support to `bun` for those scripts but did make no attempt to add such tests, in fact #49241 was the PR which added these types of tests to railties which was sort of the inspiration for this change.

I want to say what we're doing here is _enough_ test coverage, and don't feel motivated to go and add additional test coverage to each of these generators. Open to suggestions here.

> Also, if we're not actually running the installers, then running Bundler is no longer necessary, and we should change `run_generator_and_bundler` to `run_generator` for performance.

It seems the main difference between `run_generator` is calling `generator.invoke_all` (which you added in 945347f97c1eb16a9b690d45c5f3d3739a6e4a47), so I've thought of getting rid of `run_generator_and_bundler` with the following:

```diff
   def test_css_option_with_cssbundling_gem
-    run_generator_and_bundler [destination_root, "--css=postcss"]
+    generator [destination_root], ["--css=postcss"]
+    run_generator_instance
```

Reply:
@jonathanhefner Ping, just wanted to see if you had a chance to look at this again, I'm still not sure the best path forward here. But for example, if we really want to keep these tests, we could split them out into an integration test that only runs on main?